runtime/src/main/scala/akka/grpc/GrpcServiceException.scala

```diff
@@ -4,8 +4,38 @@
 
 package akka.grpc
 
+import java.lang.{ Iterable => jIterable }
+
+import akka.grpc.internal.GrpcExceptionHelper
+import akka.http.javadsl.model.{ HttpHeader => jHttpHeader }
+import akka.http.scaladsl.model.HttpHeader
 import io.grpc.Status
 
-class GrpcServiceException(val status: Status) extends RuntimeException(status.getDescription) {
+class GrpcServiceException(val status: Status, val headers: Seq[HttpHeader])
```

Comment:
Seq is immutable or mutable depending on whether you're on Scala 2.12 or 2.13. It might be nicer to explicitly use `immutable.Seq` or perhaps go for `List`.

Reply:
Updated to List.